sklearn/impute/_iterative.py

```diff
@@ -111,13 +111,17 @@ class IterativeImputer(_BaseImputer):
         many features with no missing values at both ``fit`` and ``transform``
         time to save compute.
 
-    min_value : float, default=None
-        Minimum possible imputed value. Default of ``None`` will set minimum
-        to negative infinity.
+    min_value : float or array-like, default=None
+        Minimum possible imputed value. Broadcast to shape (n_features,) if
+        scalar. Expects shape (n_features,) if array-like, one min value for
+        each feature. Default of ``None`` will set minimum to negative infinity
+        for all features.
 
-    max_value : float, default=None
-        Maximum possible imputed value. Default of ``None`` will set maximum
-        to positive infinity.
+    max_value : float or array-like, default=None
```

Comment:
```suggestion
    max_value : float or array-like of shape (n_features,), default=None
```

Reply:
Where do we need to put this entry about what's new?